app/controllers/api/users.php

```diff
@@ -663,6 +665,47 @@ function createUser(string $hash, mixed $hashOptions, string $userId, ?string $e
         $response->dynamic($user, Response::MODEL_USER);
     });
 
+App::put('/v1/users/:userId/labels')
```

Comment:
We have something similar for buckets where we update an array of strings. I wonder if we should have a dedicated endpoint vs having another attribute for creating and updating users.

Reply:
@eldadfux, sorry I don't quite understand. Would you please clarify what's the similar thing for buckets?

Reply:
This matches the existing APIs where we have separate APIs for updating name, email, etc. Do you recall why it was done that way?

Reply:
I synced with Eldad and we may want to introduce an Update User API in the future, but, for now, we'll just use this specific endpoint since it matches the other update user endpoints.